inst/htmlwidgets/datatables.js

```diff
@@ -499,9 +499,9 @@ HTMLWidgets.widget({
               // first, make sure the slider div leaves at least 20px between
               // the two (slider value) span's
               $x0.width(Math.max(160, $span1.outerWidth() + $span2.outerWidth() + 20));
-              // then, if the input is really wide, make the slider the same
-              // width as the input
-              if ($x0.outerWidth() < $input.outerWidth()) {
+              // then, if the input is really wide or slider is vertical,
+              // make the slider the same width as the input
+              if ($x0.outerWidth() < $input.outerWidth() | data.vertical) {
```

Comment:
```suggestion
              if ($x0.outerWidth() < $input.outerWidth() || data.vertical) {
```

I think it should be `||`, correct?

Reply:
That looks better to me as `|` is bitwise or